deployment/terraform/info.go

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2019-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package terraform
+
+import (
+	"fmt"
+)
+
+// Info displays information about the current load-test deployment.
+func (t *Terraform) Info() error {
+	output, err := t.getOutput()
+	if err != nil {
+		return err
+	}
+
+	t.displayInfo(output)
+
+	return nil
+}
+
+func (t *Terraform) displayInfo(output *terraformOutput) {
+	fmt.Println("==================================================")
+	fmt.Println("Here is the setup information:")
```

Comment:
Smaller detail but since we are keeping everything pretty compact (and I like it) I think a `Deployment information:` would be enough.